cmd/cmdutils.go

```diff
@@ -170,3 +173,48 @@ func printUnmountedStorage(client *occlient.Client, applicationName string) {
 	}
 	fmt.Println("")
 }
+
+// GetAppName returns application name
+func getAppName(client *occlient.Client, projectName string) string {
+
+	if Application != "" {
+
+		_, err := application.Exists(client, Application, projectName)
+		if err != nil {
+			checkError(err, "")
+		}
+		return Application
+	}
+	applicationName, err := application.GetCurrent(projectName)
+	checkError(err, "unable to get current application")
+
+	return applicationName
+}
+
+// GetProjectName returns project name
+func getProjectName(client *occlient.Client) string {
+	// Namespace is `--project` flag
+	if Namespace != "" {
+		_, err := project.Exists(client, Namespace)
+		if err != nil {
+			checkError(err, "")
+		}
+		client.Namespace = Namespace
+		return Namespace
+	}
+	client.Namespace = project.GetCurrent(client)
+	return project.GetCurrent(client)
+
+}
+
+func addProjectFlag(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&Namespace, "project", "p", "", "Project, Defaults to active project")
```

Comment:
We need a little bit more descriptive help message.
Also there probably shouldn't be a capital letter after comma